docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -135,6 +135,7 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/packagemanager.admin` | platform |  | The application can install and uninstall application packages. |
 | `http://tizen.org/privilege/packagemanager.clearcache` | public |  | The application can clear other applications' caches. |
 | `http://tizen.org/privilege/packagemanager.info` | public |  | The application can retrieve detailed application package information. |
+| `http://tizen.org/privilege/permission.check` | platform |  | The application can read the permission statuses of other apps. |
```

Comment:
Isn't it better "applications" than "apps"?

Reply:
I agree. Fixed.